.travis.yml

```diff
@@ -48,23 +48,23 @@ script:
   - DOCKER_CMD="docker run --rm --env-file /tmp/env_file -e GH_TOKEN -v $PWD:/git-restapi/ lmazuel/swagger-to-sdk"
   - >-
     if [[ $MODE == 'python' ]]; then
-      $DOCKER_CMD Azure/azure-sdk-for-python -v
```

Comment:
Is removing the verbose option too aggressive here, @lmazuel? I still want to be able to see warnings and errors.

Reply:
Yes, without verbose mode doesn't show anything

Reply:
I mean, yes you will get warnings and error, but no info. If everything is fine, you likely have no message at all.